bugs/security/index.md

```diff
@@ -1,12 +1,35 @@
 ---
 layout: default
-title: Known Security Bugs
+title: Security Bugs
 permalink: /bugs/security/
-crumb: Known Security Bugs
+crumb: Security Bugs
 ---
 
 {% include toc-hide.html %}
 
-### List of Known Security Bugs
+### Filing A Security Bug
+If you have found a potential security related bug [please follow the instructions here.](https://webrtc.org/bugs/).
 
-* (None so far)
+### Tracking WebRTC Chromium - Security Bugs
+Publicly visible security bugs in the Chromium issue tracker normally become
+visible 14 weeks after they are fixed and can be found with the following query:
+[WebRTC Chromium Bug-Security Tracker](https://bugs.chromium.org/p/chromium/issues/list?can=1&q=component:Blink%3EWebRTC%20Type=Bug-Security%20&sort=-modified&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified)
+
+If you have an account with the bug tracker you can set up a saved query to
+immediately notify you if a new security bug is posted.
+
+### Tracking WebRTC Standalone - Security Bugs
+Only a subset of the WebRTC codebase is used in Chrome. For standalone
+applications such as mobile applications that depend on WebRTC native fixes
+will be announced separately on [discuss-webrtc](https://groups.google.com/group/discuss-webrtc).
+
+Please be sure to subscribe to the mailing list to stay up to date.
+
+### Applying Security Patches - Standalone Clients
+Note that we will generally NOT merge security fixes backwards to any branches,
```

Comment:
See my comment below. lkgr gets security fixes, and there are no other branches in webrtc codebase.

Reply:
Most commonly people use chrome builds as a tag.